activiti-core/activiti-engine/src/test/java/org/activiti/engine/test/api/repository/ProcessDefinitionQueryTest.java

```diff
@@ -338,4 +338,8 @@ public void testQueryWithNullArgs() {
       .isThrownBy(() -> repositoryService.createProcessDefinitionQuery().processDefinitionKeys(null));
   }
 
+  public void testQueryWithEmptyIdSet() {
+    repositoryService.createProcessDefinitionQuery().processDefinitionIds(new HashSet<>(0)).list();
```

Comment:
Could you add an assertion to your test, please?
It could be simply `assertThat(...).isNotEmpty()`.

Reply:
> Could you add an assertion to your test, please?
> It could be simply `assertThat(...).isNotEmpty()`.

OK. will do it now.

Reply:
@erdemedeiros done. running ci.